modules/op-about-podtemplate.adoc

```diff
@@ -0,0 +1,53 @@
+// This module is included in the following assemblies:
+// * about/understanding-openshift-pipelines.adoc
+
+[id="about-podtemplate_{context}"]
+= Pod templates
+
+Optionally, you can define a _pod template_ in a `PipelineRun` or `TaskRun` custom resource (CR). You can use any parameters available for a `Pod` CR in the pod template. When creating pods for executing the pipeline or task, {pipelines-shortname} sets these parameters for every pod.
+
+For example, you can use a pod template to make the pod execute as a user and not as root.
+
+For a pipeline run, you can define a pod template in the `pipelineRunTemplate.podTemplate` spec, as in the following example:
+
+.Example `PipelineRun` CR with a pod template
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: mypipelinerun
+spec:
+  pipelineRef:
+    name: mypipeline
+  taskRunTemplate:
+    podTemplate:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1001
+----
+
+[NOTE]
+====
+In the earlier API version `v1beta1`, the pod template for a `PipelineRun` CR was specified as `podTemplate` directly in the `spec:` section. This format is not supported in the `v1` API.
```

Comment:
I think you need another comma here. This suggests that there is a later v1beta1. If you set it off in commas, it should read like _In the earlier API version, which is v1beta1,..._
Maybe in a follow up. Not enough to stop the merge. 
```suggestion
In the earlier API version, `v1beta1`, the pod template for a `PipelineRun` CR was specified as `podTemplate` directly in the `spec:` section. This format is not supported in the `v1` API.
```